api/pom.xml

```diff
@@ -27,23 +27,23 @@
 
     <groupId>edu.wgu.osmt</groupId>
     <artifactId>osmt-api</artifactId>
-    <version>2.6.0-SNAPSHOT</version>
+    <version>3.0.0-SNAPSHOT</version>
     <description>WGU Open Skills Management Toolset</description>
 
     <properties>
-        <java.version>11</java.version>
+        <app.version>3.0.0-SNAPSHOT</app.version>
```

Comment:
This might follow the same as manually changing the version above. If we correctly use the Maven Release plugin, we don't need to force / DRY the version as a property. It can work this way, but I've seen more chaos from this than it's worth. I suggest the just let Maven be Maven and not over-help it.